### ISSUE TITLE
SARIF Viewer does not show issues even when issues show in problems pane on Import SARIF File

### ISSUE THREAD

**Issue by HenokAddis** [microsoft/sarif-visualstudio-extension#159]
I attempted to run a static analysis build and found that even though SARIFs generated successfully in a fresh folder, the SARIF Viewer does not automatically open. When I attempt to import the SARIF file directly, they populate in the problems pane but the SARIF Viewer is still blank (see image).

![image](https://user-images.githubusercontent.com/46505299/89579195-32082300-d7e8-11ea-85ec-23d0d08eeb8e.png)

**Comment by HenokAddis** [comment]
Update: When I click on an issue in the Problems Pane, the SARIF Viewer does populate with defect-specific details but not a summary or general view.

![image](https://user-images.githubusercontent.com/46505299/89580101-b313ea00-d7e9-11ea-80dc-dd1c98a45d37.png)

**Comment by Miaku** [comment]
Also confirmed that this is an issue.

**Comment by Miaku** [comment]
We can document this quirk in our internal documentation for pilot customers - we should remove the ship blocker tag for this. Customers need an option for the viewer to open by default, so it's aligned with the VSCode experience.

**Comment by GabeDeBacker** [comment]
Perhaps add a message that says something to the effect of "select an error to see details"

**Comment by HenokAddis** [comment]
Agreed - we need some indication that the SARIF Explorer does not enable you to explore multiple sarif files at once - it is a drilldown view of a single SARIF file (for now).

It is important to note that this is not at parity with VSCode.

**Comment by HenokAddis** [comment]
Feedback from Kyle Sabo further validates this issue ( @GabeDeBacker / @lgolding)

> The information shown in the Errors list is inexplicably not shown in the extension pane that shows all the other details about an issue. I keep my Errors list auto-hidden. There appears to be no way to view what the actual error is that is shown in the Sarif Explorer pane except through the Errors list.

**Comment by Miaku** [comment]
Looks like the real issue here is documentation: We need to have folks use the problems pane to select issues in VS - we don't want to pop up UI that wasn't asked for.

**Comment by HenokAddis** [comment]
@Miaku  So then if the Error List is not active/open during a VS session, should we pop that open so they know the build is running + outputs are present?

**Comment by GabeDeBacker** [comment]
@HenokAddis - Visual Studio itself has an interesting behavior. Let's take IntelliSense errors as an example. When you "single click" an IntelliSense error in the list, the editor is not navigated to the file or location where the IntelliSense error occurred. The navigation only happens on "double click".

This is the same behavior the SARIF errors exhibit.

Now, we could add a message to the SARIF viewer when it is "empty" to say something like "select an error to see details" (guess it would have to say double-click actually) but even that message would be confusing because you can show the SARIF viewer when there are no SARIF errors in the list, or there are other errors in the list that are not SARIF errors.

In my opinion this is unfortunately consistent with things like visual studios "properties" window under solution explorer which is (at least for my usage of VS) empty most of the time with no message in it.
